Tests/Get-BMDeployment.Tests.ps1

```diff
@@ -83,52 +84,78 @@ BeforeAll {
 
             [Object] $Release,
 
-            [Object] $Application
+            [Object] $Application,
+
+            [Object] $Environment,
+
+            [String] $BuildNumber,
+
+            [String] $PipelineName,
+
+            [String] $PipelineStageName,
+
+            [String] $Status

```

Comment:
Make all these parameters named the same as the `Get-BMDeployment` function so that you can just pass them straight through with the [$PSBoundParameters](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.3#psboundparameters) automatic variable like this:

```powershell
Get-BMDeployment -Session $script:session @PSBoundParameters
```